backend/core/examples/chatbot/requirements.txt

```diff
@@ -0,0 +1,5 @@
+quivr-core==0.0.7
```

Comment:
quivr-core[base] should probably work which remove  faiss, langchain-community, ... 

`pip install "quivr-core[base]"`